ci/prow/issue_tracker_config.go

```diff
@@ -0,0 +1,142 @@
+		"        Mark the issue as fresh by adding the comment `/remove-lifecycle stale`.\\n\n"+
+		"        Stale issues rot after an additional %d days of inactivity and eventually close.\\n\n"+
+		"        If this issue is safe to close now please do so by adding the comment `/close`.\\n\\n\n"+
+		"        %s\\n\\n\n"+
+		"        /lifecycle stale", r.daysToStale, r.daysToRot, feedbackNote)
+	r.generateJob(jobName, filter, updatedTime, comment)
+
+	jobName = fmt.Sprintf("ci-%s-issue-tracker-rotten", repoForJob)
+	// Do not look at issues that has frozen or rotten label. Only look at stale labelled issues
+	filter = `
+        -label:lifecycle/frozen
+        label:lifecycle/stale
+        -label:lifecycle/rotten`
+	updatedTime = fmt.Sprintf("%dh", r.daysToRot*24)
+	comment = fmt.Sprintf("--comment=Stale issues rot after %d days of inactivity.\\n\n"+
+		"        Mark the issue as fresh by adding the comment `/remove-lifecycle rotten`.\\n\n"+
+		"        Rotten issues close after an additional %d days of inactivity.\\n\n"+
+		"        If this issue is safe to close now please do so by adding the comment `/close`.\\n\\n\n"+
+		"        %s\\n\\n\n"+
+		"        /lifecycle rotten", r.daysToRot, r.daysToClose, feedbackNote)
+	r.generateJob(jobName, filter, updatedTime, comment)
+
+	jobName = fmt.Sprintf("ci-%s-issue-tracker-close", repoForJob)
+	// Do not look at issues that has frozen label. Only look at rotten labels
+	filter = `
+		-label:lifecycle/frozen
+		-label:lifecycle/stale
+        label:lifecycle/rotten`
+	updatedTime = fmt.Sprintf("%dh", r.daysToClose*24)
+	comment = fmt.Sprintf("--comment=Rotten issues close after %d days of inactivity.\\n\n"+
+		"        Reopen the issue with `/reopen`.\\n\n"+
+		"        Mark the issue as fresh by adding the comment `/remove-lifecycle rotten`.\\n\\n\n"+
+		"        %s\\n\\n\n"+
+		"        /close", r.daysToClose, feedbackNote)
+	r.generateJob(jobName, filter, updatedTime, comment)
+}
+
+// generateJob generates a single issue tracker prow job
+func (r repoIssue) generateJob(jobName, labelFilter, updatedTime, comment string) {
+	var data periodicJobTemplateData
+	data.Base = newbaseProwJobTemplateData("knative/test-infra")
+	data.Base.ExtraRefs = append(data.Base.ExtraRefs, "  base_ref: "+data.Base.RepoBranch)
+	data.Base.Image = githubCommenterDockerImage
+	data.PeriodicJobName = jobName
+	data.CronString = periodicCron
+	data.Base.Command = jobCmd
+
+	// TODO(Fredy-Z): replace "r.name:knative/test-infra" with "org:knative" after syncing up with the WGs.
+	data.Base.Args = []string{
+		fmt.Sprintf(`--query=repo:%s
+        is:issue
+        is:open
+        %s`, r.name, labelFilter),
+		"--updated=" + updatedTime,
+		"--token=/etc/housekeeping-github-token/token",
+		comment,
+		"--template",
+		"--ceiling=10",
+		"--confirm",
+	}
+	addVolumeToJob(&data.Base, "/etc/housekeeping-github-token", "housekeeping-github-token", true, "")
+	addMonitoringPubsubLabelsToJob(&data.Base, data.PeriodicJobName)
```

Comment:
Do we really need to monitor the issue tracking jobs?

Reply:
I am not sure. @Fredy-Z  told me to add them :)

Reply:
removed the issue tracker labels